WordPress/Classes/ViewRelated/Views/WPContentAttributionView.m

```diff
@@ -102,6 +105,14 @@ - (void)configureConstraints
                                                                  options:NSLayoutFormatAlignAllLeft
                                                                  metrics:metrics
                                                                    views:views]];
+    [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"|-(-9)-[_readItLaterIndicatorImageView(labelHeight)]"
+                                                                 options:0
+                                                                 metrics:metrics
+                                                                   views:views]];
+    [self addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-(-9)-[_readItLaterIndicatorImageView(labelHeight)]"
```

Comment:
Good idea. I'm going to move it but I'll make sure to make the visual format use a constant in the new place.